server/src/cm_cmd_exec.h

```diff
@@ -73,44 +73,174 @@
 #endif
 #endif
 
+#include <vector>
+#include "cm_autojob.h"
+
 typedef enum
 {
     CUBRID_MODE_CS = 0,
     CUBRID_MODE_SA = 1
 } T_CUBRID_MODE;
 
-typedef struct
-{
+struct SpaceDbVolumeInfoOldFormat{
     int volid;
-    int total_page;
-    int free_page;
+    int total_size;
+    int free_size;
+    int data_size;
+    int index_size;
     char purpose[16];
```

Comment:
Should use define or constants instead of literals for size.
For vol_name and location -> PATH_MAX, if they would store file or path names.